scripts/update-fields-in-cosmos.js

```diff
@@ -0,0 +1,96 @@
+const {
+  establishConnectionToCosmos,
+  closeCosmosConnection
+} = require("../src/connectors/cosmos.client");
+const {
+  getRegistrationByFsaRn
+} = require("../src/connectors/registrationDb/registrationDb");
+const { connectToDb, closeConnection } = require("../src/db/db");
+const { logEmitter } = require("../src/services/logging.service");
+
+let beCache;
+let recordsUpdated = [];
+let recordsFailedToUpdate = [];
+
+const updateFieldsInCosmos = async () => {
+  beCache = await establishConnectionToCosmos("registrations", "registrations");
+
+  const records = await beCache
+    .find({ collected: { $exists: false } })
+    .toArray();
+  logEmitter.emit(
+    "info",
+    `Updating fields of ${records.length} records in Cosmos`
+  );
+
+  await connectToDb();
+
+  const promises = records.map(async (rec) => {
+    await updateFields(rec);
+  });
+  await Promise.allSettled(promises);
+  logEmitter.emit(
+    "info",
+    `Successfully updated fields of ${recordsUpdated.length} records in cosmos: ${recordsUpdated}`
+  );
+  logEmitter.emit(
+    "info",
+    `Failed to update fields of ${recordsFailedToUpdate.length} records in cosmos: ${recordsFailedToUpdate}`
+  );
+};
+
+const updateFields = async (rec) => {
+  // Grab fields for each record from their corresponding registration in PG
+  const registration = await getRegistrationByFsaRn(rec["fsa-rn"]);
+  if (!registration) {
+    logEmitter.emit("info", `No registration found in PG for ${rec["fsa-rn"]}`);
+    return;
+  }
+
+  const {
+    collected,
+    collected_at,
+    createdAt,
+    updatedAt,
+    direct_submission
+  } = registration.dataValues;
+
+  logEmitter.emit("info", `Updating BE Cache FSA-RN: ${rec["fsa-rn"]}`);
+  try {
+    await beCache.updateOne(
+      { "fsa-rn": rec["fsa-rn"] },
+      {
+        $set: {
+          collected: collected,
+          collected_at: collected_at,
+          createdAt: createdAt,
+          updatedAt: updatedAt,
```

Comment:
createdAt/updatedAt are built-in postgres fields, so not sure if we need to transfer them over, as new records won't have these field as I don't believe they get manually set anywhere?